README.md

```diff
@@ -13,6 +13,11 @@ the producer and 3 million msgs/second for the consumer.
 
 **librdkafka** is licensed under the 2-clause BSD license.
 
+KAFKA is a registered trademark of The Apache Software Foundation and
+has been licensed for use by librdkafka. librdkafka has no
+affiliation with and is not endorsed by The Apache Software Foundation.
```

Comment:
as per my PRs on the bindings, i'd be inclined to make a new section "License" further down and put this text and the license in that. i think you want higher impact stuff at the top.